### DISCUSSION TITLE
a nice and useful suggestion, the ability to turn up or down volume of left or right channels for users in the volume dialog

### DISCUSSION THREAD

**Discussion by peter1384** [BearWare/TeamTalk5#2332] [Ideas]
currently, we have mute left and mute right checkboxes for users in the volume dialog, which is really useful. When i was reading the codes of how the android client mutes a user's voice when you turn on the mute voice swich, and i understood it sets left channel and right channel to 0 as soon as we toggle this swich on. So, I understood that it's possible in qt client to add a left channel and right channel sliders, to be able to for example, turn someone's left channel a bit down, and another person's right channel a bit up, if it makes you more comfurt, or it's useful for recordings and stuf
so, please add the left and right sliders to the volume dialog, raather than a slider that turns both.
Do you like this idea?

**Comment by Simon818** [comment]
I think this would be a step backwards, as we'd need to adjust two different sliders for a simple volume change. If someone is too quiet, I'd need to turn both the left and right sliders up by 10%, and that's just a lot of unnecessary clunking around. However, I would love to see a pan slider (left/right balance) which defaults to 50%. It would bring back some of the functionality of the 3D positioning system. I would use this on all platforms.
